main.tf

```diff
@@ -16,13 +21,13 @@ resource "random_password" "password" {
 }
 
 resource "local_file" "current_work_directory" {
-    filename = "${local.working_dir}/.ansible"
+  filename = "${local.working_dir}/.ansible"
 }
 
 resource "local_file" "copy_kubeconfig" {
-    content = file("${local.working_dir}/kube_config_rancher-cluster.yml")
-    filename = "${var.working_dir}/kube_config_rancher-cluster.yml"
-    depends_on = ["null_resource.ansible_playbook"]
+  content    = file("${local.working_dir}/kube_config_rancher-cluster.yml")
+  filename   = "${var.working_dir}/kube_config_rancher-cluster.yml"
+  depends_on = ["null_resource.ansible_playbook"]
 }
```

Comment:
I had trouble with both `local_file.current_work_directory` and `local_file.copy_kubeconfig` not properly respecting state. What do you think about making both local-exec provisioners on the `ansible_playbook` resource?

Reply:
Oh, do you still see this even with this change? Before I would expect that because the `timestamp` always changes on reach run. However, now the directory path doesn't change so I thought the state would be respected.

Reply:
Hmm I might not have actually tested this change separately. Let me try.

Reply:
It was this weird chicken-and-egg thing where the ansible_playbook could be provisioned, but it doesn't explicitly depend on the working directory? So if somehow the resources got out of sync I received an error about this file not being able to be loaded. Granted this might all be my issues messing around locally, but it feels a bit brittle.